lib/components/viewers/pattern-row.js

```diff
@@ -72,7 +72,7 @@ export default class PatternRow extends Component {
 
           {/* expansion chevron button */}
           <div className='expansion-button-container'>
-            <button className='expansion-button' onClick={this._toggleExpandedView}>
+            <button aria-label='View all departures' className='expansion-button' onClick={this._toggleExpandedView}>
```

Comment:
`aria-label` should rather say "Expand|Collapse all departures for route xxx" depending on `this.state.expanded` so that users of screen readers can know what that state is and what route they are touching.

Reply:
The `aria-expanded` and `aria-controls` attribute should be used too, see https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions. (`aria-controls` is the id of the element which will be shown/hidden by actuating the button.)

Reply:
And maybe a `role` attribute can be set to the `VelocityTransitionGroup` at L82...

Reply:
Have expanded `aria-label`, added `aria-expanded` and `aria-controls` in bc710192935bea3619dca1522de31788edee0a49, but am not sure what role you're thinking of for the `VelocityTransitionGroup`

Reply:
aria attributes added in bc710192935bea3619dca1522de31788edee0a49, roles added in 34da5db712af0cddc7242c81e64b00f04a752aeb